src/lib/scripts/voting-gauge.generator.ts

```diff
@@ -193,7 +194,7 @@ async function getPoolInfo(
   retries = 5
 ): Promise<VotingGauge['pool']> {
   log(`getPoolInfo. poolId: network: ${network} poolId: ${poolId}`);
-  const subgraphEndpoint = config[network].subgraph;
+  const subgraphEndpoint = subgraphFallbackService.url.value;
```

Comment:
I think this one needs to be kept as before because it won't be possible to query the decentralized subgraph from the CLI when running the generator script.

Reply:
fixed